### PR TITLE
feat:- added rating, toast,.grid,.slider

### PR DESCRIPTION
**Pull request by 05him**
Added rating, toast, grid, and slider components in the component library

live URL:- https://05him.github.io/Dark_UI/

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Looks good

File: components/components.css [hunk 1]
```diff
@@ -44,6 +47,8 @@
   --card-overlay-bg: rgba(0, 0, 0, 0.3);
 
   --input-error: rgb(255, 0, 0);
+
```
Comment (vashnavichauhan1825) [inline components/components.css:50]: avoid extra spacing

**Review by Megha-Joshi — APPROVED** [review]
Good to merge

File: components/components.css (hunk 1 above)
Comment (05him) [reply components/components.css:50]: thank you for the response, but I've given spaces over there to differentiate between the colors of components
like 
(all the colors of alerts) (space of one line)(all colors of cards)

**Pull request merged**